rca/urls.py

```diff
@@ -10,11 +10,17 @@
 from wagtail.documents import urls as wagtaildocs_urls
 from wagtail.utils.urlpatterns import decorate_urlpatterns
 
+from rca.account_management.views import CustomLoginView
 from rca.utils.cache import get_default_cache_control_decorator
 from rca.wagtailapi.api import api_router
 
+WAGTAIL_FRONTEND_LOGIN_TEMPLATE = getattr(
+    settings, "WAGTAIL_FRONTEND_LOGIN_TEMPLATE", "wagtailcore/login.html"
+)
 # Private URLs are not meant to be cached.
 private_urlpatterns = [
+    path("admin/login/", CustomLoginView.as_view(), name="wagtailcore_login"),
+    path("admin/_util/login/", CustomLoginView.as_view(), name="wagtailcore_login"),
```

Comment:
I have no idea why I need to define both of these, but I do.

Reply:
Strange that they're both allowed to have the same `name="wagtailcore_login"`, maybe one is used by Django itself and the other by Wagtail...?